src/telegram/ext/_utils/networkloop.py

```diff
@@ -82,15 +87,29 @@ async def network_retry_loop(
 
             * < 0: Retry indefinitely.
             * 0: No retries.
-            * > 0: Number of retries.
+            * > 0: Number of retries
+
+            Must be negative if `infinite_loop` is set to True..
```

Comment:
Do we need to honour this? Cant we just ignore it/set it to -1 when infinite_loop is set? If you feel like it we can always raise an error then but tbh I would just do it behind the scenes.

And we can change True to :obj:`True` and remove the double .

Reply:
I'd like to keep it explicit to avoid surprising behavior :) especially since this is a rather important internal helper

Reply:
Just feels like additional work, if I already set infinite_loop to true, why do I then also need to set retries to infinite.

But sure, if you think it helps.

Reply:
yea it feels redundant

Reply:
Yeah that would be better 👍🏽

Reply:
done :)